packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.Props.ts

```diff
@@ -0,0 +1,116 @@
+import { IButtonProps } from '../../Button';
+
+export interface ISwatchColorPickerProps {
+
+  /**
+   * the number of columns for the swatch color picker
+   */
+  width: number;
```

Comment:
Width is going to translate into CSS width, which is usually measured in pixels. We should probably give this a name that reflects the fact that it is a column count.